examples/tutorials/advance_model_config/02_device.ipynb

```diff
@@ -0,0 +1,489 @@
+{
```

Comment:
<p>we increasing -&gt; we increase</p><p> </p><p>we got speed up training time is reduced by 1.5 times!  - this sentence is a mess</p><p> </p><p>Training time does not decrease by 2 times because overheads appear during information exchange between multiple GPUs.  - It is unclear, which model's training time you mean</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>